### PR TITLE
199 betterify css

### PR DESCRIPTION
**Pull request by ozeliurs**

### CODE REVIEW

**Comment by Apoorva64** [comment]
# ☁️ Nx Cloud Report

CI is running/has finished running commands for commit 8a63f72f47dab796d6e2f8ecc696e9f0305e424e. As they complete they will appear below. Click to see the status, the terminal output, and the build insights.

[📂 See all runs for this branch](https://cloud.nx.app/branch?workspaceId=640ee8fb56da4d6500e89717&branchName=211,199-betterify-css)

---


<details>
<summary>✅ Successfully ran 1 target </summary>

- [`nx affected --target=lint --parallel=3`](https://cloud.nx.app/runs/OaOY1lg1LL)
</details>

---

Sent with 💌 from [NxCloud.](https://nx.app)

File: apps/front-end/front-office/src/app/components/quiz-view/quiz-results/quiz-results.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     {{ score_text }}
   </div>
   <div class="result full card" *ngIf="canScore" (click)="score()">
-    {{ score_numeric }}
+    {{ score_numeric }}/{{ state.history.length }}
```
Comment (Guekka) [inline apps/front-end/front-office/src/app/components/quiz-view/quiz-results/quiz-results.component.html:6]: That's not CSS but it's fine

File: apps/front-end/front-office/src/app/components/quiz-view/help-page/help-page.component.ts [hunk 2]
```diff
@@ -1,8 +1,49 @@
...
+          this.quiz = gameState.quiz;
+          this.currentQuizQuestion = gameState.history.length;
+
+          const textClues = gameState.currentQuestion?.clues.filter(
+            (clue) => clue.text
+          );
+
+          if (textClues && textClues.length > 0) {
+            this.randomClue =
+              textClues[Math.floor(Math.random() * textClues.length)];
```
Comment (Guekka) [inline apps/front-end/front-office/src/app/components/quiz-view/help-page/help-page.component.ts:40]: Might want to put a comment to remind Math.random() < 1 so it cannot out of bounds

File: apps/front-end/front-office/src/app/components/quiz-view/quiz-results/quiz-results.component.html (hunk 1 above)
Comment (ozeliurs) [reply apps/front-end/front-office/src/app/components/quiz-view/quiz-results/quiz-results.component.html:6]: Please put this in your .gitignore

File: apps/front-end/front-office/src/app/components/quiz-view/help-page/help-page.component.ts (hunk 2 above)
Comment (ozeliurs) [reply apps/front-end/front-office/src/app/components/quiz-view/help-page/help-page.component.ts]: https://github.com/2019-2020-ps6/2022-2023-ps6-webonjour/pull/211/commits/84ebc10ec02d0d1be9c6f63f75999fcbf9315d26

File: apps/front-end/front-office/src/app/components/quiz-view/quiz-results/quiz-results.component.ts [hunk 3]
```diff
@@ -46,26 +47,18 @@ export class QuizResultsComponent implements OnDestroy, OnInit {
...
-            .subscribe((accommodation) => {
-              this.canReplay =
-                accommodation.data.filter((accommodation) => {
-                  return accommodation.title === 'Peut recommencer le quiz';
-                }).length > 0;
+      .subscribe((accommodation) => {
+        this.canReplay =
+          accommodation.filter((accommodation) => {
+            return accommodation.title === 'Peut recommencer le quiz';
+          }).length > 0;
```
Comment (Guekka) [inline apps/front-end/front-office/src/app/components/quiz-view/quiz-results/quiz-results.component.ts:56]:
```suggestion
          accommodation.some((accommodation) => {
            return accommodation.title === 'Peut recommencer le quiz';
          })

File: apps/front-end/front-office/src/app/components/quiz-view/quiz-results/quiz-results.component.ts [hunk 4]
```diff
@@ -46,26 +47,18 @@ export class QuizResultsComponent implements OnDestroy, OnInit {
...
-                  return (
-                    accommodation.title === 'Affiche le score à la fin du quiz'
-                  );
-                }).length > 0;
-            });
-        }
+        this.canScore =
+          accommodation.filter((accommodation) => {
+            return accommodation.title === 'Affiche le score à la fin du quiz';
+          }).length > 0;
```
Comment (Guekka) [inline apps/front-end/front-office/src/app/components/quiz-view/quiz-results/quiz-results.component.ts:61]: Same as above

**Review by Apoorva64 — APPROVED** [review]

**Pull request merged**